docs-beta/package.json

```diff
@@ -34,5 +36,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "docusaurus": "^1.14.7"
```

Comment:
A bit confused here - shouldn't this be 2.0+? I know it's beta, but let's use v2 for this.

Reply:
Since "the default doc site functionalities provided by Docusaurus 1 are now provided by @docusaurus/preset-classic", and we have @docusaurus/preset-classic as a dependency, I just deleted the devDependencies.